### PR TITLE
Change add movement button

### PR DESCRIPTION
**Pull request by asharonbaltazar**
I've decided to remove the button from the top menu and implement it in a floating action button. It scales well on mobile, even on smaller screens, and adds a bit of color to the otherwise spartan UI. Desktops don't adapt, however, and thus another solution should be found.

While typing this, I realized the original issue of button ambiguity hasn't been addressed. I tweaked the floating button to include the description of the action. I've included both, with the original one commented out.

### CODE REVIEW

**Comment by Drvanon** [comment]
From your own writing and from the images you sent I think we agree that the FAB solution is nicer. It's 1:30am here right now. Tomorrow I'll review your code.

**Comment by Drvanon** [comment]
Something I just noticed is that your pull request is to master, within Gridt we use master to represent the current state of deployment and api-development to gather all changes that get deployed in one go.

File: gridt/src/app/movements/movements.page.html [hunk 1]
```diff
@@ -1,73 +1,77 @@
...
+      <ion-row>
+        <ion-col size="12" size-sm="8" offset-sm="2" text-center>
+          <p>There are no movements right now, please come back later!</p>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
     <ion-grid *ngIf="movements$ | async">
-
-  <!-- Create a list item for each search result.
+      <!-- Create a list item for each search result.
```
Comment (Drvanon) [inline gridt/src/app/movements/movements.page.html:32]: Thank you for looking out for code quality, much appreciated! I personally don't prefer the way this comment got aligned.

File: gridt/src/app/movements/movements.page.html [hunk 2]
```diff
@@ -1,73 +1,77 @@
...
+  </ion-fab>
 
-</ion-list>
-
+  <!-- Regular floating button
+    <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+      <ion-fab-button color="primary" routerLink="/add">
+        <ion-icon name="add"></ion-icon>
+      </ion-fab-button>
+    </ion-fab> -->
```
Comment (Drvanon) [inline gridt/src/app/movements/movements.page.html:76]: I personally prefer this one. If we wanted to go for absolute clarity we could implement something like: 
press once => show add movement text => press twice go to add movements page
but that smells like over engineering to me.

File: gridt/src/app/movements/movements.page.scss [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
-    font-size: 12px;
-}
\ No newline at end of file
+  font-size: 12px;
+}
+
+.custom {
+  width: 10rem;
+  --border-radius: 30px;
+}
```
Comment (Drvanon) [inline gridt/src/app/movements/movements.page.scss:8]: This won't be necessary when implementing the normal button. (Reminder to remove.)

**Review by Drvanon — COMMENTED** [review]
I like it, it is more understandable this way I think. I do recognize that on the desktop version the FAB is behaving a bit strangely (the button hovers over the scrolling bar) as you can see in the picture below. Do you think you could fix that?
pic-selected-200908-1122-17

**Comment by asharonbaltazar** [comment]
@Drvanon hello, and thank you for examining my code.

Regarding the push to master, that must've been a mistake — I'm sure I created a new branch just before committing, but should've double-checked. I will fix that.

Regarding the formatting, I'll disable my automatic formatter before committing — you're the not first one to complain.

**Comment by Drvanon** [comment]
Actually I do like the formatting, it was just that particular instance where it did not work out very well. On the branch stuff: you can change the base branch by clicking the edit button on this page in the top right corner.

**Comment by asharonbaltazar** [comment]
So please enlighten me, as I'm not yet familiar with PR etiquette:

All the things you've mentioned above are reminders for the development team or for me?

**Comment by Drvanon** [comment]
In the future, please do create your branch name with reference to the issue you are solving. For now that does not matter sufficiently to really do it. What did matter was changing the branch to api-development, but I see you have already done that, so I we can leave that.

**Comment by Drvanon** [comment]
Much more relevant are the comments I left on your code. Once you have addressed them I will merge the pull request.

**Comment by asharonbaltazar** [comment]
As a beginner, I'm having trouble with the correct procedures. I've addressed your comments. Please have a look.

**Comment by Drvanon** [comment]
Why did you close this pull request? I was planning to look at your code still.

**Comment by asharonbaltazar** [comment]
Sorry, I want to organize the commits. It's very messy

**Comment by Drvanon** [comment]
We generally work squash commits into the api-development branch, precisely because of this reason. There are several git beginners in our community so we have opted for the method that allows the easiest integration of work. Though if you feel like you want to organize your code for your own purposes you are more than welcome to.

**Pull request closed without merging**